qemu/tests/virtual_nic_private.py

```diff
@@ -69,16 +95,20 @@ def data_mon(session, cmd, timeout):
             if_func = utils_net.get_windows_nic_attribute
             args = (mon_session, "macaddress", mon_macaddr, "netconnectionid")
         interface_name = if_func(*args)
-        tcpdump_cmd = tcpdump_cmd % (addresses[1], addresses[0],
-                                     interface_name)
+        if params.get("os_type") == "linux":
+            tcpdump_cmd = tcpdump_cmd % (addresses[1], addresses[0],
+                                         interface_name)
+        else:
+            tcpdump_cmd = tcpdump_cmd % (addresses[1], addresses[0])
         dthread = utils_misc.InterruptedThread(data_mon,
                                                (sessions[2],
                                                 tcpdump_cmd,
                                                 mon_process_timeout))
 
         logging.info("Tcpdump mon start ...")
         logging.info("Creating %dMB file on guest1", filesize)
-        sessions[0].cmd(dd_cmd % (src_file, filesize), timeout=timeout)
+        sessions[0].sendline(dd_cmd % (src_file, filesize))
+        time.sleep(60)
```

Comment:
@xiywang Would you help explain why change it like this? Cause this way we won't check if the command return as expected.

Reply:
@vivianQizhu While using the previous code in testing, I found that the case hangs with these kind of lines. That leads to timeout error. Enlarge timeout value will not help.
As long as this cmd is meant for dd file, I think it will be fine not check the return status. Because in following lines, the 'dd'ed file will be transferred and if the file is not exists, error will be announced.

Reply:
@xiywang Em..that's interesting, how reproducible of it? And does it occurred only on Windows or both OSs?

Reply:
@vivianQizhu Only on windows guests. 3/5 can hit this issue on my host.